doc/source/whatsnew/v0.18.1.txt

```diff
@@ -124,6 +124,7 @@ Bug Fixes
 - Bug in numpy compatibility of ``np.round()`` on a ``Series`` (:issue:`12600`)
 - Bug in ``Series`` construction with ``Categorical`` and ``dtype='category'`` is specified (:issue:`12574`)
 - Bugs in concatenation with a coercable dtype was too aggressive. (:issue:`12411`, :issue:`12045`, :issue:`11594`, :issue:`10571`)
+- Bug in `float_format` option with option not being validated as callable. (:issue:`12706`, :issue:`12711`)
```

Comment:
double back-ticks on ``float_format``, only list the issue option, not the PR one (e.g. 12706)